Dockerfile

```diff
@@ -1,8 +1,15 @@
-FROM python:3.5-alpine
+FROM alpine:latest
 
-RUN apk add --update jq
+RUN apk update && apk upgrade
+# sqlite is not used, and has vulns
+RUN apk del sqlite 
+RUN apk add python3
 
-RUN pip install --upgrade pip
-RUN pip install --upgrade awscli
+# get the latest straight from the source - upstream version has known vulns
+RUN wget https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 \
```

Comment:
Maybe one day github will let us grab the latest release. Today is not that day. 😞